akka-persistence/src/main/scala/akka/persistence/Eventsourced.scala

```diff
@@ -17,24 +17,24 @@ import scala.collection.immutable
 import scala.concurrent.duration.FiniteDuration
 import scala.util.control.NonFatal
 
-/**
- * INTERNAL API
- */
+/** INTERNAL API */
+@InternalApi
 private[persistence] object Eventsourced {
   // ok to wrap around (2*Int.MaxValue restarts will not happen within a journal roundtrip)
-  private val instanceIdCounter = new AtomicInteger(1)
+  private[akka] val instanceIdCounter = new AtomicInteger(1)
```

Comment:
Just remove all the `privates` the surrounding object is already `private[persistence`.

Reply:
Trying to keep changes minimal, this one is not needed in any case